cmd/chartsvc/models/chart.go

```diff
@@ -62,3 +60,10 @@ type ChartFiles struct {
 	Readme string
 	Values string
 }
+
+// ChartLatest express the latest version for a chart in a repo
+type ChartLatest struct {
+	Chart  string `json:"chart"`
+	Latest string `json:"latest"`
+	Repo   string `json:"repo"`
```

Comment:
Couldn't we have their proper types in this struct? It's difficult to know in this case which one is a Chart, is it `Chart`? Is it `Latest`?

Reply:
Update:

I see now, I'd rename those fields to make them more explicit.

Chart => Name
Latest => LatestVersion
Repo => RepositoryName

Reply:
okay

Reply:
okay